src/pages/selectJourneyDirection.tsx

```diff
@@ -0,0 +1,145 @@
+                                <h1 className="govuk-fieldset__heading" id="page-heading">
+                                    Please select your journey direction
+                                </h1>
+                            </legend>
+                            <div className="govuk-!-margin-top-5">
+                                <FormElementWrapper
+                                    errors={errors}
+                                    errorId={outboundErrorId}
+                                    errorClass="govuk-radios--error"
+                                >
+                                    <div>
+                                        <span className="govuk-hint" id="outbound-journey-selection-hint">
+                                            Outbound Journey
+                                        </span>
+                                        <DirectionDropdown
+                                            journeyPatterns={service.journeyPatterns}
+                                            selectNameID="outboundJourney"
+                                            outboundJourney={outboundJourney}
+                                        />
+                                    </div>
+                                </FormElementWrapper>
+                            </div>
+                            <div className="govuk-!-margin-top-6">
+                                <FormElementWrapper
+                                    errors={errors}
+                                    errorId={inboundErrorId}
+                                    errorClass="govuk-radios--error"
+                                >
+                                    <div>
+                                        <span className="govuk-hint" id="inbound-journey-selection-hint">
+                                            Inbound Journey
+                                        </span>
+                                        <DirectionDropdown
+                                            journeyPatterns={service.journeyPatterns}
+                                            selectNameID="inboundJourney"
+                                            inboundJourney={inboundJourney}
+                                        />
+                                    </div>
+                                </FormElementWrapper>
+                            </div>
+                        </fieldset>
+                    </div>
+                    <input
+                        type="submit"
+                        value="Continue"
+                        id="continue-button"
+                        className="govuk-button govuk-button--start"
+                    />
+                </form>
+            </main>
+        </Layout>
+    );
+};
+
+export const getServerSideProps = async (ctx: NextPageContext): Promise<{}> => {
+    deleteCookieOnServerSide(ctx, JOURNEY_COOKIE);
```

Comment:
Do we need to delete this on the server side? Other pages aren't doing this, generally, so I'm unfamiliar with why we would do it.

Reply:
this was taken from direction page not sure why it is being done there also. I have removed it from mine